includes/Feed/FeedProductFormatter.php

```diff
@@ -0,0 +1,217 @@
+<?php
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use SkyVerge\WooCommerce\Facebook\Products;
+
+/**
+ * Class responsible for formatting product for feed file.
+ */
+class FeedProductFormatter {
```

Comment:
Could the product format generated by this class also be used by the batch API sync process? If so, we could rename it to a more something applicable to both sync methods e.g. `ProductSync/ProductFormatter`.

Reply:
I don't really know. Maybe it could be but this should be a separate task if we would want to do this if it makes sense. I don't think that we will touch Batch API for a long time if ever.